src/components/FilterableList/FilterableList.Styles.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components';
+import {
+  filterableListBackgroundColorLight,
+  filterableListBackgroundColorDark
+} from '../../common/cssVariables';
+
+export const StyledUl = styled.ul`
+  margin: 0;
+  list-style-type: none;
+  padding: 0 0.5em;
+  background-color: ${({ theme }) =>
+    theme.isLight ? filterableListBackgroundColorLight : filterableListBackgroundColorDark};
```

Comment:
Maybe consider creating a theme object with the current colors based on the current mode and get the colors from the theme object here so that you don't have to repeat this logic everywhere?